src/cli/index.js

```diff
@@ -102,6 +102,7 @@ function useLocalInstallation () {
     var local = resolveCwd('testcafe/lib/cli');
 
     if (local && local !== __filename) {
+        log.write('Using locally installed version of TestCafe.');
```

Comment:
@inikulin  Just to be sure, is this line should be there? (I have this question because the line is not related to assertions)